core/dbt/include/global_project/macros/materializations/configs.sql

```diff
@@ -19,3 +19,19 @@
   {% endif %}
   {% do return(config_store_failures) %}
 {% endmacro %}
+
+{% macro set_query_tag() %}
+  {{ return(adapter.dispatch('set_query_tag', 'dbt')()) }}
+{% endmacro %}
+
+{% macro unset_query_tag(original_query_tag) %}
+  {{ return(adapter.dispatch('unset_query_tag', 'dbt')(original_query_tag)) }}
+{% endmacro %}
+
+{% macro default__set_query_tag() %}
+  {{ log("Couldn't set query tag, current adapter does not have get_current_query_tag() macro implemented.",info=True) }}
+{% endmacro%}
+
+{% macro default__unset_query_tag() %}
+  {{ log("Couldn't unset query tag, current adapter does not have get_current_query_tag() macro implemented.",info=True) }}
+{% endmacro%}
```

Comment:
This is fine during development, but I think we'd want this to be an actual noop by default (= just return an empty string). No need to clutter the logs for Postgres users with a bunch of these messages.